tutorials/inter_region_reports/data/simple_region_iperf.yaml

```diff
@@ -1,14 +1,10 @@
 # iperf benchmark for throughput - simple 1:1 zone test.
 iperf:
   flag_matrix: inter_region
-  flag_matrix_filters:
-    # For each zone:extra_zone pair where zone != extra_zone.
-    inter_region: "zones < extra_zones"
   flag_matrix_defs:
     inter_region:
       gce_network_tier: [premium]
-      zones: [us-central1-a,us-east1-b]
-      extra_zones: [us-central1-a,us-east1-b]
+      zone: [[us-central1-a,us-east1-b]]
```

Comment:
Well, this looks better. Is there a way to do this without the flag matrix? A list in flags perhaps, or does that not work?